files-community/Files
# Fix: When a ZIP file is opened via file association, the Extract Button is not displayed on the toolbar #15457

**Resolved / Related Issues**

When a ZIP file is opened via file association, the Extract Button is not displayed on the toolbar.
- Closes #15457

**Steps used to test these changes**

1. Open a compressed file with Files.
2. See the extract button showing in the AddressToolbar

**Note**
I have tried to make changes in how CanExtract works. Let me know any suggested changes or fixes I should do.
